### PR TITLE
Drop Py2.7 and Py3.5 from public CI

### PR DESCRIPTION
**Pull request by sklam**
as titled.

### CODE REVIEW

**Comment by seibert** [comment]
there is no Python 3.6 build for NumPy 1.10.  Can you raise it to NumPy 1.11?

**Comment by seibert** [comment]
Github won't let me leave comments outside the diff range, but you also need to update `TEST_COUNT` in `azure-pipelines.yml` to `18`.

**Comment by sklam** [comment]
so glad the turn around time is so much faster now

**Comment by sklam** [comment]
Hm... maybe i should also get rid of np<1.15

File: azure-pipelines.yml [hunk 1]
```diff
@@ -12,9 +12,9 @@ jobs:
...
     matrix:
-      py27_np16:
-        PYTHON: '2.7.16'
-        NUMPY: '1.16'
+      py38_np117:
+        PYTHON: '3.8'
+        NUMPY: '1.17'
         CONDA_ENV: 'travisci'
         TEST_START_INDEX: 0
       py37_np16:
```
Comment (stuartarchibald) [inline azure-pipelines.yml]: This should be `py37_np116`.

Comment (sklam) [reply azure-pipelines.yml]:
```suggestion
      py37_np116:
```

**Comment by stuartarchibald** [comment]
Thanks for the fix.

**Pull request merged**